src/indexer/index_writer.rs

```diff
@@ -461,6 +461,15 @@ impl IndexWriter {
         self.segment_updater.garbage_collect_files()
     }
 
+    /// Clear the index
+    /// Enables users to rebuild the index,
+    /// by clearing and resubmitting necessary documents
+    /// Garbage collects unused files after removing segments
```

Comment:
Clear -> Clears

Add a line about the fact that this operation, like all others requires `commiting`.

> Garbage collects unused files after removing segments

This operation removes all existing segments from the index.
The files associated to these segments will be garbage collected upon `.commit()`.

Reply:
The typo is still here.